toolbox/deps_update/util.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+)
+
+// GetMD5Hash generates an MD5 digest of the given string
+func GetMD5Hash(text string) string {
+	hash := md5.Sum([]byte(text))
+	return hex.EncodeToString(hash[:])
+}
+
+// Shell run command on shell and get back output and error if get one
+func Shell(format string, args ...interface{}) (string, error) {
```

Comment:
my suggestion here is maybe don't bother with writing a helper unless you need the extra logging

rather than doing the `fmt.Sprintf`  with the formatting you can just use the underlying signature and pass in the executable and list of args right? seems funky that you always create the string only to split it again...


like `exec.Command("./some/script.sh", []string{"-p", "flag", "-d", fmt.Sprintf("%d", integerSomething)}).CombinedOutput()` seems nearly equivalent